tests/arc4/test_tuple.py

```diff
@@ -279,3 +289,8 @@ def _compare_abi_and_arc4_values(
         assert arc4_value.native == abi_value
     else:
         assert arc4_value.bytes == int_to_bytes(abi_value, len(arc4_value.bytes))
+
+
+def test_tuple_return_with_primitive_type(context: AlgopyTestContext) -> None:  # noqa: ARG001
```

Comment:
I copied this test into `main` branch without the fix and it was passing. Either the test wasn't triggering the bug or the bug has been fixed somehow in `main` branch???

Reply:
The error is unique to versions of python starting from 3.13 and higher https://github.com/python/cpython/issues/101162